src/kibana/components/vislib/visualizations/tile_map.js

```diff
@@ -47,29 +42,38 @@ define(function (require) {
      * @return {Function} Creates the map
      */
     TileMap.prototype.draw = function () {
+
       var self = this;
       var $elem = $(this.chartEl);
       var div;
```

Comment:
`div` is never used in this scope, and it's clobbered in the `each` call (line 74). It should be inside that iteration.